packages/matchbox/src/components/Checkbox/Checkbox.js

```diff
@@ -105,7 +129,11 @@ Checkbox.displayName = 'Checkbox';
 Checkbox.Group = Group;
 Checkbox.propTypes = {
   id: PropTypes.string.isRequired,
-  checked: PropTypes.bool,
+  checked: PropTypes.oneOf([true, false, 'indeterminate']),
+  /**
+   * 'indeterminate' does not work with defaultChecked
+   */
+  defaultChecked: PropTypes.oneOf([true, false]),
```

Comment:
Any reason to do this instead of PropTypes.bool?

Reply:
Oh good catch haha